llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -853,6 +853,15 @@ def CreateHandleFromBinding : DXILOp<217, createHandleFromBinding> {
   let stages = [Stages<DXIL1_6, [all_stages]>];
 }
 
+def WaveActiveAnyTrue : DXILOp<113, waveAnyTrue> {
+  let Doc = "returns true if the expression is true in any of the active lanes in the current wave";
+  let LLVMIntrinsic = int_dx_wave_any;
+  let arguments = [Int1Ty];
+  let result = Int1Ty;
+  let stages = [Stages<DXIL1_0, [all_stages]>];
+  let attributes = [Attributes<DXIL1_0, [ReadNone]>];
```

Comment:
```suggestion
```
There is no attribute specified for `WaveAnyTrue` in `hctdb.py`. As discussed, there are many others with incorrectly added attributes, which will be cleaned-up in https://github.com/llvm/llvm-project/issues/115912